doc/user/user/user-groups/index.md

```diff
@@ -32,75 +32,6 @@ The projects list provides a high-level overview of project details, including:
 * The date that the group was last modified, and
 * (if you are permitted) group deletion buttons.
 
-Searching the Groups Table
---------------------------
+{% include tutorials/common/managing-user-groups.md %}
 
-You can filter the user groups in IRIDA using the text-field at the top of the groups table:
```

Comment:
Looks like the filter instructions weren't copied over to managing-user-groups.md. Is there a reason for that?

Reply:
I removed them due to the instructions just detailing that there is a search bar. These instructions were not present for any of the other tables that all had search bars. Would you want me to add it back for user groups? Or add it for every table? Or just keep it removed?